Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -7,12 +7,10 @@ internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControl
 
   internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
     let backedController = BackerDashboardProjectsViewController()
-    let savedController = BackerDashboardProjectsViewController()
+    backedController.configureWith(projectsType: .backed, sort: sort)
```

Comment:
we typically use a static `configuredWith` on controllers for instantiating them and configuring them in one swoop, e.g. https://github.com/kickstarter/ios-oss/blob/master/Kickstarter-iOS/Views/Controllers/BackingViewController.swift#L35-L39